### PR TITLE
Added tmuxinator scripts, configs, and docs

### PR DESCRIPTION
**Pull request by kawaiiPlat**

### CODE REVIEW

**Comment by uf-mil-investigator** [comment]
Hello, it's your friendly InvestiGator bot here!

The **docs preview** for this PR is available at https://uf-mil.github.io/pr-docs/804.

```css
Last updated at:
► c29d20cc9e5fc8d28bde915b4d8c34148445442c
► 2022-08-20 17:26 EDT
```

Have a great day! Go gators! 🐊

File: docs/software/devtools.md [hunk 1]
```diff
@@ -55,6 +55,37 @@ $ pre-commit run --all-files
...
+$ tx stop <config> # stop a session that is running
+```
+
+The MIL configs for tmuxinator are hard linked from
+`$MIL_REPO/.tmuxinatorConfigs/<name>.yaml` to `~/.config/tmuxinator/<name>.yaml`.
+If you want, you can add personal tmuxinator configs you should add them in `.config`.
+If you need to add MIL-wide configs, place them in `.tmuxinatorConfigs`
+
+
+
```
Comment (cbrxyz) [inline docs/software/devtools.md:88]: I think this should go below Vim/Emacs in this file, and possibly below bash/zsh. Only some members use tmux regularly.

File: scripts/rc.d/tmuxinator.bash [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+#Setting for the editor to use with tmuxinator edit <config>
+export EDITOR=$(which vim)
```
Comment (cbrxyz) [inline scripts/rc.d/tmuxinator.bash:4]: Why is this added here? What if the user already has `$EDITOR` exported to another term (`nvim`, `emacs`)?

File: .tmuxinatorConfigs/testing.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# /home/$(USER)/.config/tmuxinator/testing.yml
+# Testing
```
Comment (cbrxyz) [inline .tmuxinatorConfigs/testing.yml:2]: Can you maybe add a comment explaining what this configuration does? I can't understand without needing to launch the actual configuration myself.

I'm confused on:
1. Why do we have a "testing" configuration? Does this mean "testing" as in "testing at Lake Wauburg", "testing that `tmuxinator` works", or an accidental test file that was left in this PR?
2. What is this config file supposed to accomplish? Is this a config file that is ready to go to be used all the time, or should we just launch this file once to show that the program works?

Comment (kawaiiPlat) [reply .tmuxinatorConfigs/testing.yml:2]: 1. This is a base file, probably better called example.
2. As per above, basically just showing how to create a simple setup. This would need to be extended by Alex or you with whatever tools you want available.

File: scripts/rc.d/tmuxinator.bash (hunk 2 above)
Comment (kawaiiPlat) [reply scripts/rc.d/tmuxinator.bash:4]: It is needed only for editing the configs using the tmuxinator commands, which removes the need for a user to determine the location of the config files themselves. It can have a check to see if editor is already set

**Review by cbrxyz — APPROVED** [review]
Thanks, Jarrod! This looks great!
